src/components/report-problem-issue-type/styles.js

```diff
@@ -3,12 +3,27 @@ import { StyleSheet } from 'react-native';
 
 export default StyleSheet.create({
 
-transparentWrapper: {
-    flex: .12,
-    backgroundColor: 'transparent',
-    flexDirection:'row' ,
-    alignItems:'center',
-    justifyContent:'center',
-    borderTopWidth:1
+ 
+largeimage: {
+	flex:1,
+	borderWidth:1
+},
+midText: {
+	flex:.1	,
+	textAlign:'center'
 },
+iconBarWrapper: {
+	justifyContent:'center',
+	paddingTop:50,
+	flex:1
+},
+iconBar: {
+	flex: 1,
```

Comment:
Made consistent for that and all other styles.js files.  Also changed the sublime indent to 2 instead of 4 spaces.